stable/metrics-server/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.3.2
 description: Metrics Server is a cluster-wide aggregator of resource usage data.
 name: metrics-server
-version: 2.8.1
+version: 2.8.2
```

Comment:
Accidental addition?

Reply:
yeah, I have no idea how that happened... looking into how to get rid of it now.

Reply:
Cool 🙂